src/tile-board/Tileboard.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import Tile from "./Tile";
+
+function TileBoard() {
+  const [arrayGrid] = useState(() => createArrayGrid());
+
+  return (
+    <div className="container text-center mt-5">
+      {arrayGrid.map((row, key) => {
+        return <div key={key}>{row}</div>;
+      })}
+    </div>
+  );
+}
+
+/**
+ * Generates an initial square grid of tiles
+ */
+function createArrayGrid() {
+  const allTiles = [];
+  for (let i = 0; i < 5; i++) {
+    const row = [];
+
+    for (let j = 0; j < 5; j++) {
+      row.push(<Tile key={`${i}, ${j}`} />);
```

Comment:
since this arrayGrid is being used as state, I would use different values to represent the grid items. boolean values (like `false` for light off, `true` for light on) should work. 

I think the Tile component definition would be better moved into the `Tileboard` return: maybe line 10, in place of `return <div key={key}>{row}</div>;`.